docs/aerodromes/classc/Williamtown.md

```diff
@@ -21,6 +21,65 @@ WLM ADC owns the airspace within the WLM MIL CTR A (`SFC`-`A050`). This may be a
   <figcaption>WLM ADC Airspace</figcaption>
 </figure>
 
+### SID Selection
+Aircraft planned via **KEXAR**, **LIMLO**, **MONDO**, **UBSON**, shall be assigned the **Procedural SID** that terminates at the appropriate waypoint.  
+Aircraft who are not planned via those points or who are negative RNAV may be assigned a RADAR SID or a visual departure. 
+
+#### Coded Departures (Fast Jets)
+Visual Departures are commonly requested by high performance military jets and often processed in the form of a coded departure (e.g. CLASSIC1 departure), which can be found in the [FIHA AD2 Supps](https://ais-af.airforce.gov.au/australian-aip){target=new} for YWLM. These coded departures provide a corridor with altitude constraints to join the Military Training Areas east and west of YWLM, avoiding civilian traffic inbound to YSSY. 
+
+| Tracking Gate | Departure | Initial Constraint | Route |
+| ----- | ---------| --------------- | ------------------------ |
+| G1-G2 | PEPPER 1 | `A080` at ERVEV | ERVEV G1/G2 (Transition) |
+| G3-G7 | CLASSIC 1| `F125` at G3-G7 |
+| G9    | RADAR    | `F125` at G9    |
+| UKDIP | CLASSIC 1| `A090` at UKDIP |
+| OLVEP | CLASSIC 1| `F130` at OLVEP |
+
```

Comment:
@ST6-Bravo  looks like we're missing the rest of the route cells in the table. Also, is there any chance you might be able to source/create a diagram so our controllers can visualise the procedures?